src/IssueCreator/IssueCreator.csproj

```diff
@@ -30,7 +30,7 @@
     <PublisherName>Alex Ghiondea</PublisherName>
     <WebPage>publish.htm</WebPage>
     <AutorunEnabled>true</AutorunEnabled>
-    <ApplicationRevision>16</ApplicationRevision>
+    <ApplicationRevision>17</ApplicationRevision>

```

Comment:
If you don't want to have to remember this, look at the Nerdbank.GitVersioning nupkg Andrew Arnott publishes. I use that for a lot of projects. Easy to use and forget about.

Reply:
I will definitively do that! I was going to look for a way to automate this :).